docs/get-started/getting-started-alluxio.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: page
+title: Alluxio
+nav_order: 5
+parent: Getting-Started
+---
+# Get Started with RAPIDS with Alluxio
+
+RAPIDS plugin can remarkably accelerate the computing part of the whole SQL query by
+leveraging GPUs, but it’s hard to accelerate the data reading process when the data
+is in the cloud file system because of network overhead. Alluxio is an open source
+data orchestration platform that brings your data closer to compute across clusters,
+regions, clouds, and countries for reducing the network overhead. This guide will
+go through how to set up the RAPIDS Accelerator for Apache Spark with Alluxio with
+a on premise cluster.
+
+## Prerequisites
+
+Assuming that user has successfully setup and ran the RAPIDS Accelerator with on premise
+cluster according to [this doc](getting-started-on-prem.md)
+
+This guide will deploy Alluxio on Yarn cluster with 2 NodeManagers and 1 ResourceManager,
+and describe the instructions to configure Swiftstack as Alluxio’s under storage system.
+
+Let's assume the hostnames of Yarn cluster are respectively
+
+``` json
+RM_hostname
+NM_hostname_1
+NM_hostname_2
+```
+
+## Alluxio setup
+
+It is recommending to deploy Alluxio workers in each NodeManager and Alluxio master in
+ResourceManager.
```

Comment:
we may want to put the workers on the nodemanager so they are on the same nodes as the tasks will run, the master can go anywhere, we pick ResourceManager for convenience.  I don't think there is any reason to have it on the same host is there?